runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineOptions.java

```diff
@@ -120,7 +120,6 @@
   void setDataflowServiceOptions(List<String> options);
 
   /** Run the job as a specific service account, instead of the default GCE robot. */
-  @Hidden
```

Comment:
Does this coincide with the service account that would be used on GCE?

If so then this LGTM otherwise I believe the intent was to not propagate this value to the worker (the purpose of `@Hidden`) so there wasn't a mismatch.

Reply:
The idea being that you would use `@JsonIgnore` instead of `@Hidden` if we still don't want to make this show up on the worker but show up in the display data.

Reply:
Yes, this is the service account that would be used by Dataflow workers (and it was shown in UI before the change https://github.com/apache/beam/pull/14970).

This option (--serviceAccount) is advertised under "Specifying a user-managed worker service account" here: https://cloud.google.com/dataflow/docs/concepts/security-and-permissions#worker-service-account

Reply:
Sounds good.